recipe/meta.yaml

```diff
@@ -6,49 +6,79 @@ package:
   version: {{ version }}
 
 source:
-  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/model-index-{{ version }}.tar.gz
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
     sha256: 2f9870200f3b00813881b07b90d2c67291663534e43915c75fe476b6977bf2ad
+    patches:
+      - 0001-write-deps-from-missing-file.patch
   - url: https://raw.githubusercontent.com/paperswithcode/model-index/main/LICENSE
     sha256: addec7ad58188c4d6df6f578fb0c6877bd799d787ecbd9f133213de995b345a4
```

Comment:
```suggestion
```
Now that you pull the repo, you could use the license from it.

Reply:
right